WordPress/Sniffs/WhiteSpace/CastStructureSpacingSniff.php

```diff
@@ -15,21 +19,13 @@
  * @package WPCS\WordPressCodingStandards
  *
  * @since   0.3.0
+ * @since   0.xx.0 This sniff now extends the Squiz_Sniffs_WhiteSpace_CastSpacingSniff class it was based upon.
+ * @since   0.xx.0 This sniff now has the ability to fix the issues it flags.
```

Comment:
I left it like that when I submitted the PR as 0.10.0 was nearing release but not released yet, so it was unclear whether this would go in then or later. As 0.10.0 has been released now, I'll update these to read 0.11.0.